.ci/scripts/drive_examples_win32.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
TIL our windows bots have bash, sounds like I can do some cleanup :-)

Reply:
This wouldn't actually work on the Windows bot presumably (they run it with `bash foo.sh`), but I kept it for consistency with the existing bash scripts in the repo.